shell/browser/api/electron_api_app.cc

```diff
@@ -1132,11 +1132,17 @@ void App::DisableHardwareAcceleration(gin_helper::ErrorThrower thrower) {
         "before app is ready");
     return;
   }
-  if (content::GpuDataManager::Initialized()) {
+
+  if (content::GpuDataManager::Initialized())
     content::GpuDataManager::GetInstance()->DisableHardwareAcceleration();
-  } else {
-    disable_hw_acceleration_ = true;
-  }
+  disable_hw_acceleration_ = true;
```

Comment:
This flag is used elsewhere to determine that disabling hw acceleration has been requested: https://github.com/electron/electron/blob/ee82128f5be5ab46738b1d54ce6b00828d84e2a4/shell/browser/api/electron_api_app.cc#L632

Reply:
reverted!